.changeset/orange-garlics-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mds-core/mds-ingest-service": minor
```

Comment:
I think this should be `patch`, right?

Reply:
Hmm yeah I suppose. Db migrations are weird to me for versioning. Migrations don't really seem backwards compatible to me, so I leaned towards minor.

Happy to update though.

Reply:
aha, yeah I was thinking of it as a bugfix rather than a migration. I agree db migrations are weird for versioning - I only really consider API changes in semver. Maybe that's wrong though.